src/main/webapp/app/exercises/modeling/shared/modeling-explanation-editor.component.ts

```diff
@@ -16,7 +16,7 @@ export class ModelingExplanationEditorComponent {
     explanationChange = new EventEmitter();
 
     // Add tab to the value of textarea instead of moving to the next element in DOM
-    onTextEditorTab(editor: HTMLTextAreaElement, event: KeyboardEvent) {
+    onTextEditorTab(editor: HTMLTextAreaElement, event: Event) {
```

Comment:
Just making sure here that you're sure it should be `Event` instead of `KeyboardEvent`

Reply:
Yes. This is also the same fix as #3118.